docs/user-manual.adoc

```diff
@@ -100,6 +100,7 @@ Sarah White <https://twitter.com/carbonfray[@carbonfray]>; Dan Allen <https://gi
 :uri-asciidoctorjs: {uri-org}/asciidoctor.js
 :uri-irc: irc://irc.freenode.org/#asciidoctor
 :uri-asciidoc-leveloffset: http://asciidoc.org/userguide.html#X90
+:asciidoctor-pdf-docs-ref: https://github.com/asciidoctor/asciidoctor-pdf/blob/master/README.adoc
```

Comment:
To follow the new pattern we are establishing, this attribute would be defined as:

```
:uri-asciidoctor-pdf-guide:  https://github.com/asciidoctor/asciidoctor-pdf/blob/master/README.adoc
```

I can change it after I merge. Just letting you know for future edits...and looking for your input as well.